CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
 ## Changelog
 
-####v1.0.0
+###v1.0.1
```

Comment:
You need spaces between the hashtag and the v or else the markdown doesn't format correctly.